index.html

```diff
@@ -332,28 +323,34 @@ <h4 title='In random order'>Amazing alumni</h4>
 								<a href="https://cs.unc.edu/~csturton/">Cynthia Sturton</a>
 							</li>
 							<li>
-								<a href="http://www.inwyrd.com/">Kurt Thomas</a>
+								<a href="https://www.inwyrd.com/">Kurt Thomas</a>
 							</li>
 							<li>
 								<a href="https://notyetsecure.com/">Chris Thompson</a>
 							</li>
 							<li>
 								<a href="https://matthias.vallentin.net/">Matthias Vallentin</a>
 							</li>
-							<li>Jason Waddle</li>
+							<li>
+								<a href="https://www.linkedin.com/in/jason-waddle-3157b145/">Jason Waddle</a>
+							</li>
 							<li>
 								<a href="https://www.eecs.berkeley.edu/~daw/">David Wagner</a>
 							</li>
 							<li>
 								<a href="https://www.joelweinberger.us/">Joel Weinberger</a>
 							</li>
 							<li>
-								Alma Whitten
+								<a href="https://www.linkedin.com/in/alma-whitten-28997711/">Alma Whitten</a>
 							</li>
 							<li>
 								<a href="http://zesty.ca/">Ka-Ping Yee</a>
 							</li>
 
+							<li>
+                                                                <a href="https://people.eecs.berkeley.edu/~wzheng/">Wenting Zheng</a>
+                                                        </li>
```

Comment:
This is a **tiny** nit, but here and in a few other places the `</li>` is misaligned with the initial `<li>`. I just have my editor autoformat the file, so if yours does this too, you might want to do that when you're making the other edits. If not, no worries, happy to fix it myself! :-)